integration_tests/mongodb_test.go

```diff
@@ -246,6 +250,71 @@ var _ = Describe("Broker Compose Integration", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
+			By("inserting data for backup test", func() {
+				conn.EnsureSafe(&mgo.Safe{W: 2, FSync: true})
```

Comment:
I'm a bit concerned about the reverted test in db50116927c5851a059ae44be8026b0dbabfccc9. Have we asked Compose about the behaviour? Did you test any other write concern settings?

It looks like the backups are taken from a separate node in the cluster. I presume that `WMode: "majority"` would make no difference because there's still no guarentee that it's been written to the backup node. But I wonder if `W: 3` would?

Albeit, we'd be hardcoding some details about Compose. It'd probably be worth extracting the value to a constant like `mongodbNodesIncludingBackup` to make it clearer.

Reply:
We tried several options (sleep, explicit fsync and EnsureSafe), but probably not all of them (e.g. not with W:3). We felt that we already spent enough time on this, so wrapped it up.
We didn't ask support as from the MongoDB documentation it seemed that this behaviour is somewhat expected  with an eventually consistent cluster. But it might be worth dropping them an email to see how backup works exactly.
Also I removed this code block for now as it has no purpose.

Reply:
I opened a Compose ticket about this, but it's in my own GDS Compose account.

Reply:
Response from Compose:
> There's no set time that it takes for a write on the primary to make it to the hidden secondary where backups are taken. Generally it would be a few minutes, but it's all dependent on the replication lag that's as a result of the current level of writes that are taking place on the cluster.

> One could indeed use write concern in the drivers, however that's going to have two bad results: First is that all writes will take a very long time in your application, and second is that if one member goes down for any reason, your application will fail since it's demanding that all secondaries acknowledge a write. Backups take the secondary offline to snapshot it and pull data, so you're guaranteeing that your application will see downtime at least once a day for the duration of a backup.

> Generally speaking, don't expect backups do have up-to-the-second data that your primary has.